cmd/snap/cmd_snap_op.go

```diff
@@ -270,6 +271,7 @@ func (mx modeMixin) asksForMode() bool {
 type cmdInstall struct {
 	channelMixin
 	modeMixin
+	Revision string `long:"revision" description:"Install the given revision (only for own snaps)"`
```

Comment:
That one of yours sounds nice:

`"Install the given revision of a snap to which you must have developer access"`

Prefer that slightly as it implies it will break otherwise. The "if" in the follow ups have a small feeling of "ignored otherwise".